test/jest.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+
+  clearMocks: true,
+
+  collectCoverageFrom: ['client/src/**/*.{js,jsx,mjs}'],
+
+  coverageDirectory: '../coverage',
+
+  moduleFileExtensions: ['js', 'json', 'jsx'],
+
+  setupFiles: ['<rootDir>/enzyme.config.js'],
+
+  //setupFilesAfterEnv: ["../node_modules/jest-enzyme/lib/index.js"],
```

Comment:
Please remove comments before pushing to master.

Reply:
Comment removed, thanks for catching!